src/containers/Tabs/Admin.tsx

```diff
@@ -31,18 +43,20 @@ const AdminTabContainer = ({ isOpen }) => {
         updateTransaction,
         deleteTransaction,
         openToast
-    } = useTab(admins, identifier => ({ address: identifier }));
+    } = useTab(admins, (identifier: string) => ({ address: identifier }));
 
     const { drizzle } = drizzleReactHooks.useDrizzle();
 
-    const { addAdmin, removeAdmin } = drizzle.contracts.Admin.methods;
+    const { addAdmin, removeAdmin } = drizzle.contracts.Admin.methods as { 
+        addAdmin: (value: string) => TransactionObject<any>, 
```

Comment:
As above parameterise over never where we aren't using the value and don't understand it